nautobot/extras/templates/extras/job_list.html

```diff
@@ -7,61 +7,74 @@
             $(this).children("i").toggleClass("mdi-chevron-right mdi-chevron-down");
         });
     </script>
+    <script>
+	$(".accordion-toggle").click(function() {
+	    $(this).toggleClass("mdi-chevron-down mdi-chevron-right");
+        });
+    </script>
 {% endblock %}
 
 {% block content %}
     <h1>{% block title %}Jobs{% endblock %}</h1>
     <div class="row">
         <div class="col-md-12">
             {% if jobs %}
-                <table class="table table-hover table-headings reports">
-                    <tr>
-                        <th>Module / Job</th>
-                        <th>Description</th>
-                        <th class="text-right">Last Run</th>
-                        <th style="width: 40px"></th>
-                        <th style="width: 100px">Last Status</th>
-                        <th style="width: 180px">Last Results</th>
-                    </tr>
+	        <div class="panel-group" id="accordion">
+		    <div class="row panel">
```

Comment:
nit: looks like some tab characters snuck in here and various locations below - mind fixing them back to spaces?